QuizApp/QuizSession/Models/Answer/AnswerDataSourceModel.swift

```diff
@@ -0,0 +1,11 @@
+struct AnswerDataSourceModel {
+
+    let answer: String
+    let id: Int
```

Comment:
id is usually the first defined property.
```suggestion
    let id: Int
    let answer: String
```